src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -765,6 +767,50 @@ TableOwner(Oid relationId)
 
 
 /*
+ * SchemaOwner returns the name of the owner of the specified schema.
+ */
+char *
+SchemaOwner(Oid schemaId)
```

Comment:
It seemed like a utility function but making it static until someone needs that makes sense.